src/layouts/default.vue

```diff
@@ -1,17 +1,16 @@
 <template>
-  <div class="app grid h-screen overflow-hidden relative">
-    <div>
+  <div class="app grid relative">
+    <div class="sticky top-0 block z-40">
       <VTeleportTarget name="skip-to-content" :force-destroy="true" />
       <VMigrationNotice v-show="isReferredFromCc" />
       <VTranslationStatusBanner />
       <VHeader />
     </div>
-    <main
-      class="main embedded overflow-x-hidden"
-      :class="{ 'has-sidebar': isSidebarVisible }"
-    >
+    <main class="main embedded" :class="{ 'has-sidebar': isSidebarVisible }">
       <Nuxt ref="mainContentRef" class="min-w-0 main-page" />
-      <VSidebarTarget class="sidebar" />
+      <VSidebarTarget
+        class="sidebar fixed pb-20 right-0 h-[calc(100%-81px)] bg-dark-charcoal-06 border-s border-dark-charcoal-20 overflow-y-scroll"
```

Comment:
I hate needing to know the header height here (81px), but it's the only way to do this.

Reply:
What about the banners? They currently break the calculations

Reply:
CSS variables and calc outside of storybook would probably be a better fit than this kind of thing.

Reply:
Did you mean outside of _tailwind_, not storybook, @sarayourfriend ? 
If so, I agree with you :)

Reply:
Sorry, yes tailwind, not storybook! :sweat_smile:

Reply:
Using CSS variables instead of inline tailwind.